DXMainClient/Domain/NullDiscordHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace DTAClient.Domain
+{
+    public class NullDiscordHandler : IDiscordHandler
+    {
+        public void UpdatePresence()
+        {
+        }
+
+        public void UpdatePresence(string map, string mode, string type, string state, int players, int maxPlayers, string side, string roomName, bool isHost = false, bool isPassworded = false, bool isLocked = false, bool resetTimer = false)
+        {
+        }
+
+        public void UpdatePresence(string map, string mode, string type, string state, int players, int maxPlayers, string roomName, bool isHost = false, bool resetTimer = false)
+        {
+        }
+
+        public void UpdatePresence(string map, string mode, string state, string side, bool resetTimer = false)
+        {
+        }
+
+        public void UpdatePresence(string mission, string difficulty, string side, bool resetTimer = false)
+        {
+        }
+
+        public void UpdatePresence(string save, bool resetTimer = false)
+        {
+        }
+
+        public void Connect()
+        {
+        }
+
+        public void Disconnect()
+        {
+        }
+    }
+}
```

Comment:
I am not sure if this is a valid approach at all. Moreover it seems like it needs restart in order to recreate the handler.